sample.html

```diff
@@ -74,10 +74,22 @@
             window.pp.mb.publish(OO.EVENTS.WIKIA.SHOW_AD_FULLSCREEN_TOGGLE, false);
           }
         }, false, this);
+
+        // that's a hack for autoplay on mobile for VPAID ads
```

Comment:
We have a spreadsheet with issues. I would leave it as it is - it'll be a bit less work if we decide to fix VPAID.